### PR TITLE
Update Readme

### PR DESCRIPTION
**Pull request by brantwellman**
Updated readme with screenshot, gif, proper authorship and description.
Includes link to live site and instructions for running locally
Updated title on index.html
Updated package.json data to reflect proper repository and authorship

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,36 @@
...
+
+## Level 1
+![](http://i.imgur.com/2JlyecP.png)  
+
+## Level 2
+![](http://recordit.co/ZZaU99zS7r.gif)
+
+## Tools
+  * JavaScript / ES6
+  * [p5 JS library](https://p5js.org/) for rendering
```
Comment (afg419) [inline README.md]: This makes it sound like we're using P5's 3d functionality (if it exists) to render the game.  By far the most impressive part about this project is that this is not the case.

File: README.md [hunk 2]
```diff
@@ -1,6 +1,36 @@
-# Game Time Starter Kit
+# Space Pirates : Destroyer
 
-To install the dependencies:
+A 3D first-person shooter written in JavaScript by [Aaron Greenspan](https://github.com/afg419) and [Brant Wellman](https://github.com/brantwellman).
```
Comment (afg419) [inline README.md]: Graphics rendered with a handwritten 3D engine.

File: README.md (hunk 1 above)
Comment (brantwellman) [inline README.md]: Updated this line to make it clear we only used p5 for sketching the custom graphics.

File: README.md (hunk 2 above)
Comment (brantwellman) [inline README.md]: This is a good addition...It's in there now.

**Comment by afg419** [comment]
Cool looks good

**Pull request merged**